ui/imports/shared/stores/DAppsStore.qml

```diff
@@ -7,6 +7,8 @@ QObject {
 
     required property var controller
 
+    readonly property var globalUtilsInst: globalUtils
```

Comment:
`readonly` still works when overridden?

Reply:
removed as well. I've started with a much larger scope and later discovered we have other tasks to handle fees and the background 😄